assets/js/components/Sectors.vue

```diff
@@ -630,10 +630,10 @@ export default Vue.extend({
         // level 2 items are hidden
         .attr("opacity", (d) => d.depth == 2 ? 0 : null )
         // and really hidden
-        .style("display", (d) => d.depth == 2 ? "none" : null )
+        .style("display", (d) => d.depth == 2 ? "none" : null);
```

Comment:
the spacing there is meant to denote how the inner stuff is part of the function definition, and not another argument.